### PR TITLE
Add devfile parameters override procedure

### PR DESCRIPTION
**Pull request by mshaposhnik**
### What does this PR do?
Added proc for overriding devfile parameters via factory URL prameters.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/13617

### CODE REVIEW

File: src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// overriding-devfile-values-using-factory-parameters
+
+[id="overriding-devfile-values-using-factory-parameters_{context}"]
+= Overriding some devfile values using factory parameters
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc]:
```suggestion
= Overriding devfile values using factory parameters
```

File: src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// overriding-devfile-values-using-factory-parameters
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc]:
```suggestion
// included in:
//
// * assembly_configuring-a-workspace-using-a-devfile.adoc
```

File: src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+[id="overriding-devfile-values-using-factory-parameters_{context}"]
+= Overriding some devfile values using factory parameters
+
+A some values of the remote devfile can be overriden using specially constructed additional factory parameters. Currently allowed sections to override values are: apiVesion, metadata, projects.
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc]:
```suggestion
Values in the following sections of a remote devfile can be overridden using specially constructed additional factory parameters:

* `apiVersion`
* `metadata`
* `projects`
```

File: src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+.Prerequisites
+* A running instance of {prod}. To install an instance of {prod}, see link:{site-baseurl}che-7/che-quick-starts/[{prod-short} quick-starts].
+* The publicly-accessible standalone `devfile.yaml` file. See link:{site-baseurl}che-7//making-a-workspace-portable-using-a-devfile/[Making a workspace portable using a Devfile] for detailed information about creating and using devfiles.
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc]:
```suggestion
* A publicly accessible standalone `devfile.yaml` file. See link:{site-baseurl}che-7//making-a-workspace-portable-using-a-devfile/[Making a workspace portable using a Devfile] for detailed information about creating and using devfiles.
```

File: src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+.Procedure
+. Execute the workspace by opening the following URL: `++https://++__<your-che-host>__/f?url=https://__<yourhosturl>__/devfile.yaml&override.<parameter.path>=<value>`
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc]:
```suggestion
. Open the workspace by navigating to the following URL: `++https://++__<che-host>__/f?url=https://__<hostURL>__/devfile.yaml&override.__<parameter.path>__=__<value>__`
```

File: src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+.Example of overriding generateName property
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc]:
```suggestion
.Example of overriding the `generateName` property
```

**Review by rkratky — CHANGES_REQUESTED** [review]

**Comment by mshaposhnik** [comment]
@rkratky Please take a look if that's looks good now, thanks.

**Review by rkratky — APPROVED** [review]
Thanks, @mshaposhnik

**Pull request merged**